py-polars/docs/source/reference/expressions/categories.rst

```diff
@@ -11,3 +11,4 @@ The following methods are available under the `expr.cat` attribute.
 
     Expr.cat.get_categories
     Expr.cat.set_ordering
+    Expr.cat.to_local
```

Comment:
I don't think this should be an expression. It seems to me  like an implementation detail we only want on the memory buffers, i.e. Series

Reply:
Okay, let's have it on Series only for now!

Done.